File: group 87/day_63/homework.js

function randomMove(){
    const rps=["rock","paper","scissors"]
    let randomnum=Math.floor(Math.random()*3)
    return rps[randomnum]
}
let humanScore=0
let robotScore=0
while (humanScore < 3 && robotScore < 3){
    let robotChoice=randomMove()
    let choice=prompt("Rock,paper,scissors: ").toLowerCase()
    console.log(`Human chose ${choice} and robot chose ${robotChoice}`)
    if (choice === robotChoice){
        console.log("Draw")
    }
    else if (choice === "scissors" && robotChoice==="paper"){
        console.log("Human won")
        humanScore++
        console.log(`Human's score is ${humanScore}`)
    }
    else if (choice === "scissors" && robotChoice==="rock"){
        console.log("Robot won")
        robotScore++
        console.log(`Robot's score is ${robotScore}`)
    }
    else if (choice === "rock" && robotChoice==="scissors"){
        console.log("Human won")
        humanScore++
        console.log(`Human's score is ${humanScore}`)
    }
    else if (choice === "rock" && robotChoice==="paper"){
        console.log("Robot won")
        robotScore++
        console.log(`Robot's score is ${robotScore}`)
    }
    else if (choice === "paper" && robotChoice==="scissors"){
        console.log("Robot won")
        robotScore++
        console.log(`Robot's score is ${robotScore}`)
    }
    else if (choice === "paper" && robotChoice==="rock"){
        console.log("Human won")
        humanScore++
        console.log(`Human's score is ${humanScore}`)
    }
}

File: group 87/day_65/homework.py
import random
question = input("Do you want to play(yes/no): ").lower()
if question.startswith("y"):
    humanscore=0
    robotscore=0
    while humanscore<3 and robotscore<3:
        humanchoice=(input("rock,paper,scissors: ")).lower()
        print(" ")
        while humanchoice not in ["rock","paper","scissors"]:
            humanchoice=(input("Please input valid value(rock,paper,scissors): ")).lower()
            print(" ")
        robotchoice=random.choice(["rock","paper","scissors"])
        print(f"Human chose {humanchoice}")
        print(f"Robot chose {robotchoice}")
        print(" ")
        if humanchoice==robotchoice:
            print("Draw,no points")
            print(" ")
            print(" ")
        elif (robotchoice=="rock" and humanchoice=="scissors")or(robotchoice=="paper" and humanchoice=="rock")or(robotchoice=="scissors" and humanchoice=="paper"):
            print("Robot won")
            robotscore+=1
            print(f"Robot: {robotscore}")
            print(f"Human: {humanscore}")
            print(" ")
            print(" ")
        else:
            print("Human won")
            humanscore+=1
            print(f"Robot: {robotscore}")
            print(f"Human: {humanscore}")
            print(" ")
            print(" ")
    if humanscore>robotscore:
        print("Congrats,you won")
    else:
        print("You lost")
